build.gradle

```diff
@@ -51,7 +51,7 @@ idea {
 }
 dependencies {
     compile("org.quartz-scheduler:quartz:2.2.3")
-    compile("org.mongodb:mongo-java-driver:3.3.0")
+    compile("org.mongodb:mongodb-driver:3.3.0")
```

Comment:
Motivation for this change:
http://mongodb.github.io/mongo-java-driver/
MongoDB java driver comes in two packages: `mongodb-driver` and `mongo-java-driver`
The latter is considered legacy and contains all the stuff bundled.
`mongodb-driver` is the preferred package, it depends on `mongodb-driver-core` and `bson`.